gluoncv/data/ucf101/classification.py

```diff
@@ -161,32 +161,20 @@ def _make_dataset(self, directory, setting):
                 clips.append(item)
         return clips
 
-    def _TSN_RGB(self, directory, offsets, new_height, new_width, new_length, is_color, name_pattern):
-
-        from ...utils.filesystem import try_import_cv2
-        cv2 = try_import_cv2()
-
-        if is_color:
-            cv_read_flag = cv2.IMREAD_COLOR
-        else:
-            cv_read_flag = cv2.IMREAD_GRAYSCALE
-        interpolation = cv2.INTER_LINEAR
-
+    def _TSN_RGB(self, directory, offsets, new_height, new_width, new_length, name_pattern):
         sampled_list = []
         for _, offset in enumerate(offsets):
-            for length_id in range(1, new_length+1):
-                frame_name = name_pattern % (length_id + offset)
-                frame_path = directory + "/" + frame_name
-                cv_img_origin = cv2.imread(frame_path, cv_read_flag)
+            for length_id in range(1, new_length + 1):
+                frame_path = os.path.join(directory, name_pattern % (length_id + offset))
+                cv_img_origin = self.cv2.imread(frame_path)
                 if cv_img_origin is None:
                     raise(RuntimeError("Could not load file %s. Check data path." % (frame_path)))
                 if new_width > 0 and new_height > 0:
-                    cv_img = cv2.resize(cv_img_origin, (new_width, new_height), interpolation)
+                    cv_img = self.cv2.resize(cv_img_origin, (new_width, new_height))
                 else:
                     cv_img = cv_img_origin
-                cv_img = cv2.cvtColor(cv_img, cv2.COLOR_BGR2RGB)
+                cv_img = cv_img[:, :, ::-1]
```

Comment:
In my experiences, cv2.cvtColor is a bit faster, is there a reason for this change?